bin/add_host_to_ssh_known_hosts

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
+
+# Parameter can be just a host name, or a full http, https or git URL.
+URL="${1:?You need to provide an URL as first parameter}"
```

Comment:
This is fancy – I had to run it locally to see how it worked! 😅

Reply:
You can thank https://devhints.io/bash 😃